protocol/0028-GOVE-governance.md

```diff
@@ -146,7 +146,9 @@ For market change proposals the network will additionally calculate
 1. `LP participation rate = SUM (equity-like share of all LPs who cast a vote)` (no need to divide by anything as equity-like share sums up to `1`).
 1. `LP for rate = SUM (all who voted for) / LP participation rate`.
 
-A market parameter change is passed only when:
+If the market that the proposal is changing is pending at the vote resolution time then only token holder votes are used. 
```

Comment:
So specifically: if a market has been accepted, enacted proposal still in opening auction?

Reply:
Yes. Clarified, will push.